pkgs/development/python-modules/python-zaqarclient/default.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  buildPythonPackage,
+  ddt,
+  fetchFromGitea,
+  jsonschema,
+  keystoneauth1,
+  openstackdocstheme,
+  osc-lib,
+  oslo-i18n,
+  oslo-log,
+  oslo-utils,
+  pbr,
+  pythonOlder,
+  requests-mock,
+  requests,
+  setuptools,
+  sphinxHook,
+  stestr,
+  stevedore,
+}:
+
+buildPythonPackage rec {
+  pname = "python-zaqarclient";
+  version = "2.7.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitea {
```

Comment:
I think we so far preferred fetching from github https://github.com/openstack/python-zaqarclient

Reply:
If I had to choose, I would prefer OpenDev as it is the only official Git repositories plaftorm for OpenStack projects but I'm ok to both or github-only as this is what we have been doing so far.